src/components/myprofile.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './myprofiles.css';
+import './myprofile.css';
+import styles from './myprofile.module.css';
+
+function Myprofile() {
+  const missionsData = useSelector((state) => state.missions);
+  const dragonsData = useSelector((state) => state.dragons);
+  const rocketsData = useSelector((state) => state.rockets);
+
+  return (
+    <div className={styles.profile_container}>
+      <div className="rocket-contain">
+        <h2>My Rockets</h2>
+        <ul className="table1">
+          {rocketsData.map((rocket) => {
+            if (rocket.reserved) {
+              return <li className="data" key={rocket.id}>{rocket.name}</li>;
+            }
+            return '';
+          })}
+        </ul>
+      </div>
+      <div className="container">
+        <h2 className="Mymish">My Missions</h2>
+        <ul className="containers">
+          {missionsData.map((mission) => {
```

Comment:
- [ ] Kindly ensure that you use ```filter()``` to render a list of all joined missions and reserved rockets as per the requirements.

Reply:
Hi @Ghiftee 
Thank you again for pointing this out.
> * [ ]  Kindly ensure that you use `filter()` to render a list of all joined missions and reserved rockets as per the requirements.
- Use filter() at this [commit](https://github.com/Lucash2022/React-Group-Project/commit/a77c6a699e93ade274cbe3629c5d80579faebc8c)